src/__tests__/lib/containers/QueryWrapperMenu.test.tsx

```diff
@@ -42,7 +37,6 @@ describe('it renders with basic functionality', () => {
     menuConfig: [{ sql, facet }],
     rgbIndex: 3,
     stackedBarChartConfiguration: {
```

Comment:
Question, do we always need to pass an empty object as a placeholder even if there is nothing in it? Thanks.

Reply:
good question!  maybe these can be removed, I'll take a look...

Reply:
removed the empty stacked bar chart configurations!

Reply:
but missed the test, hold on :)

Reply:
Apparently, if there is no stacked bar chart configuration then the stacked bar chart is not shown today.  And if it's provided (even if it's empty), it will be shown.  So, for now these need to be there.

Reply:
Got it. Sounds good.